File: School source codes/Lab 12/CompareArrays.cs
using System;
using System.Collections.Generic;
using System.Linq;

class MainClass {
  public static bool Compare(int[] x, int[] y){
    if (x.SequenceEqual(y)) return true;
    else return false;
  }
  public static void Main() {
    int size,count1=0,count2=0,test;
    Console.WriteLine("What is the array size: ");
    size = Convert.ToInt32(Console.ReadLine());
    int[] Arr1 = new int[size];
    int[] Arr2 = new int[size];
    Console.WriteLine("Enter the numbers for array 1: ");
    while (count1 < size){
      test = Convert.ToInt32(Console.ReadLine());
      Arr1[count1] = test;
      count1++;
    }
    Console.WriteLine("Enter the numbers for array 2: ");
    while (count2 < size){
      test = Convert.ToInt32(Console.ReadLine());
      Arr2[count2] = test;
      count2++;
    }
    if (Compare(Arr1,Arr2) == false)
      {Console.WriteLine("The arrays are not identical.");}
    else
      {Console.WriteLine("The arrays are identical.");}
  }
}

File: School source codes/Lab 12/CompareArrays.rb
def Compare(x, y)
  if (x == y) then true else false end
end
count1 = 0
count2 = 0
print "What is the array size: "
size = gets.chomp.to_i
Arr1 = Array.new(size)
Arr2 = Array.new(size)
print "Enter the numbers for array 1: "
while (count1 < size)
  Arr1 << gets.chomp.to_i
  count1 += 1
end
print "Enter the numbers for array 2: "
while (count2 < size)
  Arr2 << gets.chomp.to_i
  count2 += 1
end
if (Compare(Arr1,Arr2) == false)
  puts "The arrays are not identical."
else
  puts "The arrays are identical."
end